MANUAL_GCP_DEPLOYMENT.md

```diff
@@ -0,0 +1,18 @@
+# Deployment Guide for membrane-backend
+
+This guide outlines the steps to deploy manually membrane-backend to Google Cloud Run.
+
+## Steps
+ 1. Setup your GCP environment. You can use the instructions [GCP Project Setup Guide](https://github.com/ai-cfia/devops/blob/main/gcp-setup-script/gcp-project-setup-guide.md)
+
+2.  You will need to add the "Secret Manager Secret Accessor" role to your service account
+
+3. Enable "Secret Manager" service on your GCP project.
```

Comment:
is there a way to give them a deeplink to find this?

Reply:
I put a link in, it goes to the secret manager page but you still need to click on go to console and that only appears if you are logged in